### DISCUSSION TITLE
Rejected and Approved on the same day leading to "Last verified as null"

### DISCUSSION THREAD

**Discussion by 916-22k** [community/community#113385] [GitHub Education] [answered]
### Hi everyone,

_Hello, I will treat this "discussion" as if it were my question/support ticket._
In total, I had made 4 submission requests, of which all were made on the 18th of March. I had been rejected the first time for not having 2FA enabled. I had been rejected the second time because GitHub's image processing algorithm was unable to read and identify the following: [lastname, firstname, date]. I had been rejected the third time because it still couldn't read and identify my last name, and it also surprisingly figured out that it was an acceptance letter. The fourth time I applied, the status was: 
_Congrats! Your request for [@916-22k](https://github.com/916-22k) was approved and your benefits will be available soon._
This was available for me to see POST-application via the link: https://education.github.com/discount_requests/application 
Also at this page, I found the below text in the (body) rather than on the (right side :=where the 4 attempts were listed)
_Hi, 916-22k! You were last verified as a null on Mar 18, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits._ 

Is this "null" a server-side javascript problem, or an impactless client-side firefox bug?

**Comment by samus-aran** [comment, marked as answer]
Hi @916-22k  👋

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You were last verified as a "null" on...](https://github.com/orgs/community/discussions/111352#null-last-verified-on-date)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
